apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -87,15 +86,9 @@ class ReviewTab extends Component {
 
     const initialLoadPromises = [];
 
+    const onDoneFetchingComments = data => this.setState({comments: data});
     initialLoadPromises.push(
-      this.dataApi
-        .getCodeReviewCommentsForProject()
-        .done((data, _, request) => {
-          this.setState({
-            comments: data,
-            token: request.getResponseHeader('csrf-token')
-          });
-        })
+      this.dataApi.getCodeReviewCommentsForProject(onDoneFetchingComments)
```

Comment:
It's a matter of personal preference, but for a one line function like this, I often like to put it directly inline as it's clearer exactly what's happening. If you do want to keep the function separate, I'd ask for a rename describing what the function is doing, rather than why it's being called.

i.e. for the first option, something like this:
```
this.dataApi.getCodeReviewCommentsForProject(data => this.setState({comments: data}));
```

for the second option, something like this:
```
const setComments = data => this.setState({comments: data});
...
this.dataApi.getCodeReviewCommentsForProject(setComments);
```